src/ScrumBoardItBundle/Controller/DefaultController.php

```diff
@@ -31,15 +33,26 @@ public function indexAction()
     public function homeAction(Request $request)
     {
         $service = $this->get($this->getUser()->getConnector().'.api');
+        $session = $request->getSession();
 
         $searchFilters = $service->getSearchFilters($request);
         $issues = $service->searchIssues($searchFilters);
 
         $searchEntity = new SearchEntity($searchFilters);
```

Comment:
If you don't need the entity, don't keep it in memory :

$form = $this->createForm($service->getFormType(), new SearchEntity($searchFilters););